src/main/kotlin/net/atos/client/sd/exception/RuntimeExceptionMapper.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.client.sd.exception
+
+import jakarta.ws.rs.core.MultivaluedMap
+import jakarta.ws.rs.core.Response
+import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper
+
+class RuntimeExceptionMapper : ResponseExceptionMapper<RuntimeException> {
+    override fun handles(status: Int, headers: MultivaluedMap<String, Any>): Boolean =
+        status >= Response.Status.INTERNAL_SERVER_ERROR.statusCode
+
+    override fun toThrowable(response: Response): RuntimeException = RuntimeException(
+        "Server response from SmartDocuments wsxmldeposit: ${response.status} (${response.statusInfo})"
```

Comment:
Is `wsxmldeposit` always the source of this exception? Would be nice if we can pull this from the response.